timelock-server/src/suiteTest/java/com/palantir/atlasdb/timelock/MultiNodePaxosTimeLockServerIntegrationTest.java

```diff
@@ -431,6 +431,20 @@ public void stressTest() {
         }
     }
 
+    @Test
+    public void fastForwardsDoNotHaveCrossNamespaceImpact() {
+        long freshTimestamp = client.getFreshTimestamp();
+
+        long fastForwardTimestamp = freshTimestamp + 100_000_000;
+        NamespacedClients other = cluster.clientForRandomNamespace().throughWireMockProxy();
+        other.timestampManagementService().fastForwardTimestamp(fastForwardTimestamp);
+
+        cluster.failoverToNewLeader(client.namespace());
+        assertThat(client.getFreshTimestamp())
+                .isGreaterThan(freshTimestamp)
+                .isLessThan(fastForwardTimestamp);
+    }
+
```

Comment:
minor: you can add assertion that other was fastforwarded, not necessary if it's already covered by other tests.

Reply:
Good flag - this _isn't_ covered by the other tests!